__tests__/app.test.js

```diff
@@ -36,13 +36,55 @@ describe("app", () => {
         });
       });
     });
+    describe("/reviews", () => {
+      describe("/:review_id", () => {
+        describe("GET: /api/reviews/:review_id", () => {
+          test("200: responds with a review object that has given review id", () => {
+            return request(app)
```

Comment:
Good tests, you could also add another one that checks if the response is equal to the specific article you are requesting 🙂